package-lock.json

```diff
@@ -3242,6 +3242,44 @@
 				"estraverse": "4.2.0"
```

Comment:
I wouldn't expect there should be any changes to this file if we're not updating `package.json`.